dds_web/security/auth.py

```diff
@@ -112,17 +141,20 @@ def __verify_general_token(token):
     if expiration_time and (
         dds_web.utils.current_time() <= datetime.datetime.fromtimestamp(expiration_time)
     ):
-        username = data.get("sub")
-        if username:
-            user = models.User.query.get(username)
-        if user and user.is_active:
-            return user, data
-
-        return None, data
+        return data
 
     raise AuthenticationError(message="Expired token")
 
 
+def user_from_subject(subject):
+    if subject:
+        user = models.User.query.get(subject)
+    if user and user.is_active:
+        return user
```

Comment:
lines 152-153 should be indented once to the right

Reply:
Doesn't really matter since the if statement checks if user is set

Reply:
I do think it matters, I think user is not defined otherwise. I'll indent it at least.